Automattic/jetpack
# JS Component: move Product offer placeholder above button

This PR moves up the error placeholder of the ProductOffer component above the CTA button.

Fixes #

#### Changes proposed in this Pull Request:

* Product Offer: moves up the error placeholder above the CTA button.
* Alert: set a min-height for the Alter component


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Disconnect the site in case it's connected
* Go to Protect interstitial page
* Test locally (http://localhost) to reproduce an error connection
* Click on the `Get started with Jetpack Protect` button
* Confirm you see the error
* Confirm it doesn't produce a jumper effect

https://user-images.githubusercontent.com/77539/170359372-2276c9eb-35d2-4ed7-99d4-378148e06b0a.mov
